includes/reader-revenue/templates/myaccount-after-delete-account.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * My Account page for reader account deletion.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+?>
+
+<div>
+	<h1>
+		<?php echo esc_html__( 'Your account has been deleted.', 'newspack' ); ?>
+	</h1>
+	<p>
+		<?php echo esc_html__( "We're sorry to see you go. But hope you'll continue to follow our coverage.", 'newspack' ); ?>
```

Comment:
```suggestion
		<?php echo esc_html__( "We're sorry to see you go. We hope you continue to enjoy our content and consider supporting us again in the future.", 'newspack' ); ?>
```

Reply:
Maybe link `supporting us again` to the donation page? Probably won't get much conversion that way, but never hurts.

Reply:
That would mean assuming that a reader has donated before. And if someone deletes their account, it might not be the best place to pitch for financial support :D